src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,135 @@
+package duke;
+
+import java.io.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.FileWriter;
+
+public class Storage {
+
+    protected String filePath;
+
+    public Storage(String filePath) {
+        this.filePath = filePath;
+    }
+
+    /**
+     * Loads task list from file
+     *
+     * @return List of task in file
+     * @throws IOException
+     */
+    public ArrayList<Task> load() throws IOException {
```

Comment:
This method is also a bit long. Maybe you can extract some lines out into different methods, such as a method for checking the type of task to add or whether a task is done or not.